deployment/main.tf

```diff
@@ -198,6 +198,8 @@ resource "null_resource" "up" {
 
   provisioner "remote-exec" {
     inline = [
+      "docker image prune -a --filter 'until=7d'",
+      "docker container prune --filter 'until=7d'",
```

Comment:
ah c'est malin!

La CI n'a pas apprécié (`Are you sure you want to continue? [y/N]`) : `-f` ? (x2)

Reply:
Yes j'ai vu ça, c'est corrigé mais je voulais finir la PR avant de relancer.